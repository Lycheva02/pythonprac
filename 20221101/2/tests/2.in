r = Triangle((0, 0), (0, 5), (5, 0))
s = Triangle((1, 1), (2, 1), (1, 3))
t = Triangle((0, 0), (10, 10), (13, 42))
print(*(f"{n}({bool(x)}):{round(abs(x), 3)}" for n, x in zip("rst", (r, s, t))))
print(f"{s < t=}, {r < t=}, {r < s=}")
print(f"{s in t=}, {r in t=}, {s in r=}")
print(f"{r & t=}, {t & r=}, {s & r=}, {s & t=}")
