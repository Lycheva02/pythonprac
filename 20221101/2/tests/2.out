r(True):12.5 s(True):1.0 t(True):145.0
s < t=True, r < t=True, r < s=False
s in t=False, r in t=False, s in r=True
r & t=True, t & r=True, s & r=True, s & t=True
